Integrations/integration-DemistoRESTAPI.yml

```diff
@@ -0,0 +1,98 @@
+description: Use Demisto server REST APIs
+configuration:
+- display: Demisto Server URL
+  name: url
+  defaultvalue: ""
+  type: 0
+  required: true
+- display: Demisto Server API Key
+  name: apikey
+  defaultvalue: ""
+  type: 4
+  required: true
+- display: Trust certificate (insecure)
+  name: insecure
+  defaultvalue: "false"
+  type: 8
+  required: false
+- display: User system proxy settings
+  name: proxy
+  defaultvalue: "true"
+  type: 8
+  required: false
+script:
+  script: |
+    var serverURL = params.url;
+    if (serverURL.slice(-1) === '/') {
+        serverURL = serverURL.slice(0,-1);
+    }
+
+    var sendRequest = function(method, url, body) {
+        var requestUrl = serverURL;
+        if (url.slice(-1) != '/') {
+            requestUrl += '/';
```

Comment:
I would also add support in case someone provided the server url in the uri argument.

Reply:
also disagree. then you don't have api key. and it makes it more complicated.

Reply:
You did not understand the comment.
If user provide:
`url=https://demo4.demisto.com/users` and not `url=users` it will not work for them.
Even the argument name us url and not uri, you should support both options.

Reply:
changed to URI, and added example. don't think we should handle such case.
also remember this integration is the basic connection, and the real use cases will be done in scripts.